library/src-bootstrapped/scala/quoted/util/Var.scala

```diff
@@ -0,0 +1,48 @@
+package scala.quoted
+package util
+
+/** An abstraction for variable definition to use in a quoted program.
+ *  It decouples the operations of get and update, if needed to be spliced separately.
+ */
+sealed trait Var[T] {
+
+  // Retrieves the value of the variable
+  def get given QuoteContext: Expr[T]
+
+  // Update the variable with the expression of a value (`e` corresponds to the RHS of variable assignment `x = e`)
+  def update(e: Expr[T]) given QuoteContext: Expr[Unit]
+}
+
+object Var {
+  /** Create a variable initialized with `init` and used in `body`.
+   *  `body` recieves a `Var[T]` argument which exposes `get` and `update`.
+   *
+   *  `var`('(7)) {
```

Comment:
@biboudis, you forgot to adapt the comment. It should be `Var(...`. Also need to update the quote and splice syntax